scipy/special/tests/test_data.py

```diff
@@ -56,6 +56,15 @@ def data_local(func, dataname, *a, **kw):
     return FuncData(func, DATASETS_LOCAL[dataname], *a, **kw)
 
 
+# The functions lpn, lpmn, clpmn, and sph_harm appearing below are
+# deprecated in favor of legendre_p_all, assoc_legendre_p_all,
+# assoc_legendre_p_all (assoc_legendre_p_all covers lpmn and clpmn),
```

Comment:
`assoc_legendre_p_all` gets repeated doesn't it?

Reply:
I guess it's not the clearest way to say `lpmn` and `clpmn` are both covered by `assoc_legendre_p_all`, and the parenthetical covers that already, so yeah, I'll change it.

Reply:
oh, I see what you did... ok, no this is fine then!